examples/nn/mnist.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import print_function
+import argparse
+import torch
+import sys
+
+sys.path.append("../../")
+import heat as ht
+import heat.nn.functional as F
+import heat.optim as optim
+from heat.optim.lr_scheduler import StepLR
+from heat.utils import vision_transforms
+from heat.utils.data.mnist import MNISTDataset
+
+"""
+
+This file is an example script for how to use the HeAT DataParallel class to train a network on the MNIST dataset.
+To run this file execute:
+mpirun -np N python mnist.py
+where N is the number of processes, in the examples/nn/ directory
+"""
+
+
+class Net(ht.nn.Module):
+    def __init__(self):
```

Comment:
Add class doc ^^ (You could modify the docstring above a bit to fit here)

Reply:
this is just an example network, it is excluded in the docs. The text at the top is some info about the file in general for a user and is not related to the network. It will likely be added to a readme for the nn examples eventually